src/style.css

```diff
@@ -75,14 +78,16 @@ main {
   display: flex;
   flex-direction: column;
   gap: 20px;
+  font-family: 'Lato', sans-serif;
 }
 
 .add-score button {
   width: 100px;
   margin-left: 100px;
-  background-color: #fff;
+  background-color: #c96630;
   border: none;
   cursor: pointer;
   padding: 3px 2px;
-  box-shadow: 10px 5px 5px rgb(157, 154, 154);
+  box-shadow: 10px 5px 5px #cb3535;
+  font-family: 'Lato', sans-serif;
 }
```

Comment:
Kindly customize the readme file to include instructions on how to run the app. This will allow other developers be able to set up the project and run it locally.

Also, kindly remove the description under the each of the headings of the readme file.

Your readme:
![image](https://user-images.githubusercontent.com/20554583/216273563-2e3bc1a9-bb51-4425-a610-8f805b50a6bc.png)